### PR TITLE
:adhesive_bandage: Remove duplicate title in individual summaries 

### LINKED ISSUE #71: `OTC CatchUp ## Summary` is not needed twice
Each individual summary page includes the heading twice.

[Summary of CatchUp #90](https://catchup.ourtech.community/summary/90)
image of catchup summary #90

### PR DESCRIPTION
**Pull request by sreekaransrinath**
+ condense about section

Fixes #71

### CODE REVIEW

**Comment by sreekaransrinath** [comment]
Fixes #71

**Comment by sreekaransrinath** [comment]
cc @HarshKapadia2

File: summary/individual-summary.adoc [hunk 1]
```diff
@@ -12,15 +12,12 @@ v1
...
 
-NOTE: For all summaries, please visit link:https://catchup.ourtech.community/summary[catchup.ourtech.community/summary].
-
 '''
 
 == About OTC CatchUp
 
-link:https://catchup.ourtech.community[OTC CatchUps^] are weekly informal sessions involving project showcases and technical discussions. They are held every Saturday from 10:30 PM IST. link:https://catchup.ourtech.community/attend[Join in!^]
-
-NOTE: link:https://catchup.ourtech.community/showcase[Showcase a project on OTC CatchUp!^] It can be anything from a simple Calculator to an extremely complex project. We appreciate them all!
```
Comment (SirusCodes) [inline summary/individual-summary.adoc:23]: We also need this linked to individual summaries?👀

Comment (sreekaransrinath) [reply summary/individual-summary.adoc:23]: We don't, imo. Project submission is not as big of a CTA as just "join us", and it creates info clutter, drawing attention away from the most important things - the individual summary and the link to join.

Comment (SirusCodes) [reply summary/individual-summary.adoc:23]: Hmm... Makes sense.

**Comment by SirusCodes** [comment]
@tusharnankani have a look and merge of looks fine to you as well

**Comment by sreekaransrinath** [comment]
@KartikSoneji @HarshKapadia2 requesting review

**Review by tusharnankani — APPROVED** [review]
Looks alright. Thank you, Sreekaran!

**Review by KartikSoneji — APPROVED** [review]
LGTM

**Comment by KartikSoneji** [comment]
Just realised, this doesn't actually fix the duplicate title in the individual summary page.

**Comment by sreekaransrinath** [comment]
@KartikSoneji what do you mean?

**Comment by tusharnankani** [comment]
Ah damn, okay.

In #71, the issue is to remove the redundant `OTC CatchUp #90 Summary`.

**Comment by sreekaransrinath** [comment]
Bump @KartikSoneji @tusharnankani

**Comment by KartikSoneji** [comment]
> Can someone confirm that the only purpose of the combined summary template is to serve inside the individual summary page?

No. The combined summary template is also used to build, well, the combined summary.

The fix will be a bit more involved with this one, I'll leave a comment with more details on the issue.

**Pull request merged**